gr-iio/lib/CMakeLists.txt

```diff
@@ -9,15 +9,27 @@
 ########################################################################
 # Setup library
 ########################################################################
-add_library(
-    gnuradio-iio
-    attr_sink_impl.cc
-    attr_source_impl.cc
-    attr_updater_impl.cc
-    dds_control_impl.cc
-    device_sink_impl.cc
-    device_source_impl.cc
-    iio_types.cc)
+if(DEFINED LIBIIO_V1 AND NOT DEFINED CACHE{LIBIIO_V1})
```

Comment:
what does that do?

Reply:
This helped me when I was playing around with both libiio versions. I didn't always make a clean build and I added this so that gnuradio does not accidentally link to a wrong verion (maybe one what I removed from the system). This does nothing unexpected when making a clean build. If you think this is not necessay or it does not fit in with the rest of the CMakeLists from gnuradio I can take it out and leave only the `if(DEFINED LIBIIO_V1)`